packages/edit-site/src/components/sidebar-navigation-screen-template/home-template-details.js

```diff
@@ -116,14 +116,30 @@ export default function HomeTemplateDetails() {
 	 * which contains the template icon and fallback labels
 	 */
 	const templateAreas = useMemo( () => {
+		// Keep track of template part IDs that have already been added to the array.
+		const templatePartIds = [];
```

Comment:
Suggestion: This could be a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), it ensures that values are unique and has helpful methods like `.has` and `.add`.

Reply:
The `templatePart.id` isn't clientId; from TT4 examples, duplicates would be `post-meta`.

Reply:
Oh see what you mean, clever. I was just trying to cast it as a set. thanks, I'll update the PR